### ISSUE TITLE
Focus issue on Linux (KDE)

### ISSUE THREAD

**Issue by xstable** [BelledonneCommunications/linphone-desktop#551]
# To Reproduce

Linphone is open. I got an incoming call. I pick up the call and see a second window: 

![image](https://user-images.githubusercontent.com/1778068/136347077-9264554b-efde-4604-8ff5-92eb325e845b.png)

I can open other Windows (other applications) which put linphone in the background (behind those opend windows).
If I close the above windows, I can see the `two` linphone windows again.
All fine till here.
BUT, if I other windows above the linphone windows, and I click ot the icon in the Tray 
![image](https://user-images.githubusercontent.com/1778068/136347581-02351bf2-3bb0-459e-a7c1-5c34ea7b4e00.png)

Only the main-window of linphone come back to focus. The window with the current call stay's invisible.

## Expected behavior
Window of `current call` should be connected with the main-window of linphone, so that if linphone-main-window get back the focus, the `current call` window also should come back to the foreground.

In my opinion it would be a much nicer and smoother usability to not have an additional window, but put those `current call`-sections into the main-window. Maybe with a tapped pane to be able to manage multiple calls.

**Desktop (please complete the following information):**
 - OS: 5.4.148-1-MANJARO
  - Version of the SDK 4.4.19

**Comment by mbrouillet** [comment]
I totally agree on this. Finding back the current call window once a (long) call is ended, and confirming it is hung up (before saying anything unrelated to someone else) calls for a UX improvement.
Suggestions :

- integrate active call in the main window, as suggested above
- system tray icon change color when a call is active
- system tray icon click brings to foreground active call window (no matter on which desktop it is)

Thank you for this great version of Linphone. I've been searching for a good VOIP SIP linux client for more than 10 years and this is it.

**Comment by mbrouillet** [comment]
Here is my workaround : 
Put this in a local file `~/local/bin/bringuplinphone.sh `
```
#!/bin/bash
/usr/bin/wmctrl -a Appels || /usr/bin/wmctrl -a Linphone || /usr/bin/linphone
```
Run `chmod +x ~/local/bin/bringuplinphone.sh
Assign it to a key.
You may need to install wmctrl : `sudo apt install wmctrl`
